src/pages/patientView/pathwayMapper/pathwayMapper.module.scss

```diff
@@ -0,0 +1,3 @@
+.toast {
+    width: 200px;
+}
```

Comment:
We should just remove this. We don't need toasts in patient view, and we will get rid of them after fixing  cBioPortal/cbioportal#7677 anyway.